### ISSUE TITLE
Doesn't work with Cura 5.2.2

### ISSUE THREAD

**Issue by bbarenz** [smartavionics/RawMouse#23]
Get an error "The plugin RawMouse could not be loaded"...

**Comment by smartavionics** [comment]
That's probably because that Cura release uses a version of Python that is more recent than the versions supported by RawMouse.

**Comment by DivingDuck** [comment]
@smartavionics, 
FYI, the plugin version 1.2.1 is running fine on my windows 10/11 installations with Cura 5.2.2

**Comment by smartavionics** [comment]
OK, so what platform doesn't it work on?

Do you have any diagnostics? Log file or terminal output?

**Comment by bbarenz** [comment]
Yes, that is a very reasonable request and thank you for the responses. To be honest I didn't think it was being maintained so I didn't bother to troubleshoot and moved on to another solution. When you responded I went to pull the logs and I found the issue. Somehow (probably user error) I downloaded version 1.0.7. I went back to the release page to double check I had the latest version and the newest is 1.2.1. I downloaded that and it works fine. Sorry for being a lazy idiot and thank you for all your work providing this tool!
